### PR TITLE
Final renaming of activities to events

### PR DESCRIPTION
**Pull request by stevejgordon**
Please merge after PR #722 (may require me to rebase this one before accepting it)

Includes:
File and folder renaming
Final source file updates
SQL file updates
CSS class updates

This final PR fixes issue #705

### CODE REVIEW

**Comment by tonysurma** [comment]
this one can be rebased now

**Comment by tonysurma** [comment]
is it me or are there several changes that aren't really changes... Did spacing change?

File: AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Events/EditEvent.cs [hunk 1]
```diff
@@ -2,240 +2,240 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using AllReady.Areas.Admin.Features.Events;
```
Comment (tonysurma) [inline AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Events/EditEvent.cs]: seems like most/all of the changes in this file aren't real changes?

**Comment by stevejgordon** [comment]
@tonysurma I've rebased off of the latest master. I'm not sure why those files are showing complete changes. I renamed some folders which perhaps caused git to see complete file changes? I'm not really sure how to fix it though. Any thoughts?

**Comment by tonysurma** [comment]
I unfortunately don't.  @BillWagner or @MisterJames any thoughts?  Maybe it is the folder name switch?

**Comment by BillWagner** [comment]
I don't know. I hope to learn something from @MisterJames :smile:

**Comment by stevejgordon** [comment]
I'm wondering if perhaps the line endings were different in some files and when I did my renaming of folders and final manual check of the files they got updated and saved.

**Pull request merged**